lib/test_prof/cops/rspec/aggregate_failures.rb

```diff
@@ -1,170 +0,0 @@
-# frozen_string_literal: true
-
-require "rubocop"
-require "test_prof/utils"
-
-module RuboCop
-  module Cop
-    module RSpec
-      # Rejects and auto-corrects the usage of one-liners examples in favour of
-      # :aggregate_failures feature.
-      #
-      # Example:
-      #
-      #  # bad
-      #  it { is_expected.to be_success }
-      #  it { is_expected.to have_header('X-TOTAL-PAGES', 10) }
-      #  it { is_expected.to have_header('X-NEXT-PAGE', 2) }
-      #  its(:status) { is_expected.to eq(200) }
-      #
-      #  # good
-      #  it "returns the second page", :aggregate_failures do
-      #    is_expected.to be_success
-      #    is_expected.to have_header('X-TOTAL-PAGES', 10)
-      #    is_expected.to have_header('X-NEXT-PAGE', 2)
-      #    expect(subject.status).to eq(200)
-      #  end
-      #
-      class AggregateFailures < RuboCop::Cop::Cop
-        # From https://github.com/backus/rubocop-rspec/blob/master/lib/rubocop/rspec/language.rb
-        GROUP_BLOCKS = %i[
-          describe context feature example_group
-        ].freeze
-
-        EXAMPLE_BLOCKS = %i[
-          it its specify example scenario
-        ].freeze
-
-        class << self
-          def supported?
```

Comment:
Should I keep this check?

Reply:
Yeah, let's keep it (and move to `test_prof/rubocop.rb`). We do not have restrictions in the gemspec, so this is the only way to handle the unsupported version (yeah, pretty old, but still could be used).